src/main.js

```diff
@@ -33,6 +33,24 @@ function newWindow(path) {
     window.show()
   })
 
+  if (setting.get('window.maximized') === true) {
+    window.maximize()
+  }
+
+  // store the window size
+  window.on('maximize', () => {
+    setting.set('window.maximized', true)
+  })
+
+  window.on('unmaximize', () => {
+    setting.set('window.maximized', false)
+  })
+
+  window.on('resize', () => {
+    setting.set('window.width', window.getSize()[0])
```

Comment:
We should already be setting these automatically in [App.js, line 72](https://github.com/SabakiHQ/Sabaki/blob/master/src/components/App.js#L72), so I don't think setting it here again is necessary.